docs/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,30 @@
+# Issue Template
+
+* Generally speaking, issues should consist of bugs in SLEAP that will require a pull request. For general questions and suggestions for enhancements, you can start a new discussion [here](https://github.com/talmolab/sleap/discussions)
+
+## Description of issue
+
+* Please provide a clear and concise explanation of the issue
+
+## Expected behavior
+
+* Please provide a clear and concise explanation of the behavior you are expecting
+
+## Versions:
+- OS + version [e.g Linux Ubuntu 18.04]
+- Browser + version [e.g Firefox 100.0]
+- SLEAP version [e.g v1.2.3]
+- SLEAP installation method [e.g conda, pip]
```

Comment:
We can make the SLEAP installation methods check boxes with links to the method since there are relatively few out there. Ex:
- SLEAP installation method (listed [here](https://sleap.ai/installation.html#))
  - [ ] [Conda from package](https://sleap.ai/installation.html#conda-package)
  - [ ] [Conda from source](https://sleap.ai/installation.html#conda-from-source)
  - [ ] [pip package](https://sleap.ai/installation.html#pip-package)
  - [ ] [M1 Macs](https://sleap.ai/installation.html#m1-macs)